src/ui/sensor/widget_serialportsensor.ui

```diff
@@ -37,7 +37,61 @@
     </widget>
    </item>
    <item>
-    <widget class="QComboBox" name="mSerialPortComboBox"/>
+    <widget class="QComboBox" name="mSerialPortComboBox">
+     <property name="toolTip">
+      <string>Choose a serial port for the connection</string>
+     </property>
+    </widget>
+   </item>
+   <item>
+    <widget class="QLabel" name="label">
+     <property name="text">
+      <string>Baudrate</string>
+     </property>
+    </widget>
+   </item>
+   <item>
+    <widget class="QComboBox" name="mBaudrateComboBox">
+     <property name="toolTip">
+      <string>Choose a baudrate for the sensor connection</string>
+     </property>
+     <property name="currentText">
+      <string>4800</string>
+     </property>
+     <property name="currentIndex">
+      <number>0</number>
+     </property>
+     <item>
+      <property name="text">
+       <string>4800</string>
+      </property>
+     </item>
+     <item>
+      <property name="text">
+       <string>9600</string>
+      </property>
+     </item>
+     <item>
+      <property name="text">
+       <string>19200</string>
+      </property>
+     </item>
+     <item>
+      <property name="text">
+       <string>38400</string>
+      </property>
+     </item>
+     <item>
+      <property name="text">
+       <string>57600</string>
+      </property>
+     </item>
+     <item>
+      <property name="text">
+       <string>115200</string>
+      </property>
+     </item>
```

Comment:
Let's initiate the combo box items on the C++ side of things so we can attach enum values as data. 

I.e. `mBaudrateComboBox->addItem( QStringLiteral( "1200 baud" ), static_cast<int>( QSerialPort::Baud1200 ) );`

You can then use mBaudrateComboBox->currentData() to retrieve the data value, cast it back into enum. Much cleaner.

Reply:
Additionally, the baudrate setting has been set to a default of 9600 baud, just to match with the hardcoded default baudrate prior to this PR with `mBaudrateComboBox->setCurrentIndex( 3 );`